src/utils/hints/oraclesHints.js

```diff
@@ -1,4 +1,5 @@
 export const oraclesHints = {
+  oracle: 'Oracle is a component that connects Smart Contracts with real-world data. It acts as a trusted source of external information by fetching data from various sources and validating its accuracy.',
```

Comment:
I am personally more in favor of not repeating the entity name like:
```suggestion
  oracle: 'Component that connects Smart Contracts with real-world data. It acts as a trusted source of external information by fetching data from various sources and validating its accuracy.',
```

Reply:
Thanks, good improvement

Reply:
Niceee, but I also meant the other entities

Reply:
Oh, gotcha. Fixed them all